views/login.php

```diff
@@ -16,7 +16,9 @@
  * @since 0.1
  */
 include('../models/Notification.php');
+global $params;
 
+$timeout = $params[0];
```

Comment:
Check to make sure `$params` is not empty first. Otherwise you get an `Undefined offset: 0` error when going to `/login`